mongodump/prepare.go

```diff
@@ -409,13 +403,13 @@ func (dump *MongoDump) CreateIntentsForDatabase(dbName string) error {
 		if err != nil {
 			return fmt.Errorf("error decoding collection info: %v", err)
 		}
+		if dbName == "config" && dump.OutputOptions.Oplog && isReshardingCollection(collInfo.Name) {
+			return fmt.Errorf("detected resharding in progress. Cannot dump with --oplog while resharding")
+		}
```

Comment:
This code move seems intentional and significant, I think we should add a comment to make sure that the next person modifying this doesn't accidentally put it back to the original spot.

Reply:
Yeah it’s in the commit message, but a comment is a good idea, too.

Reply:
I agree, noting every skipped collection and reason for skip may be better served in another ticket where we can figure out how to best present that information as well.